### DISCUSSION TITLE
Use with quasar without vee-validate - Vue 3

### DISCUSSION THREAD

**Discussion by GabrielZulian** [dm4t2/vue-currency-input#314] [Q&A] [answered]
How can i create a simple currency component in Quasar without using vee-validate?

In examples the value of model-value comes from vee-validate, but I want to use only modelValue prop and update/emit values accordingly. Can i have an example?

**Comment by dm4t2** [comment, marked as answer]
Check this out: https://codesandbox.io/s/using-vue-currency-input-with-quasar-gwn46?file=/src/QCurrencyInput.vue
